crates/cxx-qt-gen/src/generator/rust/qobject.rs

```diff
@@ -195,9 +197,10 @@ mod tests {
             }
         };
         let parser = Parser::from(module).unwrap();
+        let structures = Structures::new(&parser.cxx_qt_data).unwrap();
 
         let rust = GeneratedRustFragment::from_qobject(
-            parser.cxx_qt_data.qobjects.values().next().unwrap(),
+            &structures.qobjects[0],
```

Comment:
would probably use `&structures.qobjects.get(0).unwrap()` here so you don't get a random index failure thing when it goes wrong